docs/swarm-api.md

```diff
@@ -24,71 +27,157 @@ POST "/images/create" : "docker import" flow not implement
 
 ## Endpoints which behave differently
 
-* `GET "/containers/{name:.*}/json"`: New field `Node` added:
+<table>
```

Comment:
There are other minor differences too, for instance volume filters aren't supported yet. Should this table include those too?